src/components/display/Dropdown.test.tsx

```diff
@@ -268,92 +238,67 @@ describe('Dropdown', () => {
 			}
 		];
 		const onClick = jest.fn();
-		render(
+		const { user } = setup(
 			<Dropdown items={dropdownItems}>
 				<Button label="opener" onClick={onClick} />
 			</Dropdown>
 		);
 
-		expect(screen.getByRole('button', { name: /opener/i })).toBeInTheDocument();
+		expect(screen.getByRole('button', { name: /opener/i })).toBeVisible();
 		// dropdown is closed
 		expect(screen.queryByText(/item/i)).not.toBeInTheDocument();
 		// open dropdown
-		userEvent.click(screen.getByRole('button', { name: /opener/i }));
+		await user.click(screen.getByRole('button', { name: /opener/i }));
 		await screen.findAllByText(/item/i);
 		// wait for listeners to be registered
-		await waitFor(
-			() =>
-				new Promise((resolve) => {
-					setTimeout(resolve, 1);
-				})
-		);
+		jest.advanceTimersByTime(TIMERS.DROPDOWN.REGISTER_LISTENER);
 		expect(screen.getByText('item 1')).toBeVisible();
 		expect(screen.getByText('item 2')).toBeVisible();
 		expect(screen.getByText('item 3')).toBeVisible();
 		expect(screen.queryByText(/tooltip/i)).not.toBeInTheDocument();
-		userEvent.hover(screen.getByText('item 1'));
-		await waitFor(
-			() =>
-				new Promise((resolve) => {
-					setTimeout(resolve, 500);
-				})
-		);
+		// wait so tooltip can register the listeners
+		jest.advanceTimersByTime(TIMERS.TOOLTIP.REGISTER_LISTENER);
+		await user.hover(screen.getByText('item 1'));
 		expect(screen.queryByText(/tooltip/i)).not.toBeInTheDocument();
-		userEvent.hover(screen.getByText('item 2'));
+		await user.hover(screen.getByText('item 2'));
 		await screen.findByText(/tooltip/i);
 		expect(screen.getByText('tooltip 2')).toBeVisible();
-		act(() => {
-			userEvent.unhover(screen.getByText('item 2'));
+		await act(async () => {
+			await user.unhover(screen.getByText('item 2'));
 		});
-		userEvent.hover(screen.getByText('item 3'));
-		await waitFor(
-			() =>
-				new Promise((resolve) => {
-					setTimeout(resolve, 500);
-				})
-		);
+		// wait so tooltip can register the listeners
+		jest.advanceTimersByTime(TIMERS.TOOLTIP.REGISTER_LISTENER);
+		await user.hover(screen.getByText('item 3'));
```

Comment:
After this hover, we need to advance timers by 500ms (create a constant TIMERS.TOOLTIP.DELAY_SHOW or similar). We need to advance timers to be sure that the tooltip is not shown.